server.js

```diff
@@ -301,20 +301,27 @@ app.get('/questionform', function(req, res) {
         return res.redirect('/');
     }
 
-    var html = questionForm();
-
-    return res.status(200).send(html);
-});
-
-app.get('/re', function(req, res){
-    var html = regexForm();
-    res.status(200).send(html);
+    return res.status(200).render('question-creation',{
+        questionType: common.questionTypes
+    });
 });
 
-app.get('/mc', function(req, res){
-    var html = multipleChoiceForm();
-    res.status(200).send(html);
-});
+app.post('/formanswer', function(req, res){
```

Comment:
this follows from my previous question. plz explain

Reply:
I think answerForm makes more sense. What do you think?

Reply:
Yeah, I can change to answerForm. That makes more sense.